src/partials/head-meta.hbs

```diff
@@ -11,10 +11,11 @@
     <meta name="feedback" content="{{{this}}}">
     {{/with}}
 
-    {{!-- Swiftype - TODO: to remove --}}
-    <meta class="swiftype" name="post-type" data-type="enum" content="{{{ or page.attributes.type 'Developer Guide' }}}">
-    <meta class="swiftype" name="neo4j-site" data-type="enum" content="Developer Site">
+    {{!-- Swiftype --}}
+    <meta class="swiftype" name="post-type" data-type="enum" content="{{{ or (or page.attributes.search-type page.attributes.type) 'Developer Guide' }}}">
```

Comment:
parenthesis are not mandatory?

```suggestion
    <meta class="swiftype" name="post-type" data-type="enum" content="{{{ (or (or page.attributes.search-type page.attributes.type) 'Developer Guide') }}}">
```

Reply:
As discussed with @recrwplay this syntax is invalid, we must only use parenthesis around function arguments.